artemis-server/src/main/resources/schema/artemis-configuration.xsd

```diff
@@ -2924,6 +2924,44 @@
       <xsd:attributeGroup ref="xml:specialAttrs"/>
    </xsd:complexType>
 
+   <xsd:complexType name="dead-letter-address-auto-create">
```

Comment:
Should be choice between this or using explicit (exisiting) shouldnt be able to set both

Reply:
Added as todo in https://jira.apache.org/jira/browse/ARTEMIS-2420.
We can't use XSD choice unfortunately in xsd all